client/src/components/ProTx/components/charts/MaltreatmentTypesPlot.js

```diff
@@ -36,78 +41,142 @@ function MaltreatmentTypesPlot({
 
     return (
       <div className="maltreatment-types-plot-layout">
-        <div className="maltreatment-types-plot-header">
-          <div className="maltreatment-types-plot-info">
-            <div className="maltreatment-types-plot-info-item">
-              <div className="maltreatment-types-plot-selected-region">
-                <span className="maltreatment-types-plot-selected-region-label">
-                  Selected {geographyLabel}
-                </span>
-                <span className="maltreatment-types-plot-selected-region-value">
-                  {fipsIdNameMaltreatment}
-                </span>
-                <span className="maltreatment-types-plot-selected-region-code">
-                  ({selectedGeographicFeatureMaltreatment})
-                </span>
-              </div>
-              <div className="maltreatment-types-plot-aggregated-count">
-                <span className="maltreatment-types-plot-aggregated-count-label">
-                  {maltreatmentLabel}
-                </span>
-                <span className="maltreatment-types-plot-aggregated-count-value">
-                  {maltreatmentTypesDataAggregateMaltreatment}
-                </span>
-              </div>
+        <div className="maltreatment-types-plot-info">
+          <div className="maltreatment-types-plot-info-region">
+            <div className="maltreatment-types-plot-selected-region">
+              <span className="maltreatment-types-plot-selected-region-label">
+                FIPS: {selectedGeographicFeatureMaltreatment}
```

Comment:
we should drop the related class names like `maltreatment-types-plot-selected-region-code` and `observed-features-plot-selected-region-code` if no longer needed.

Reply:
> I think I liked the way it was before without FIPS but not certain 🤔 . is the addition of "FIPS" just to highlight what the number is for someone who doesn't know.

Yep, because depending on what geography we are showing (eventually) these would be different types of codes and I am assuming many folks could use the reminder/help as to "what is that code", but that entire plot header section needs a reboot (I think we have a ticket to make it ao component already) so I am going to leave it for now.

Reply:
> we should drop the related class names like `maltreatment-types-plot-selected-region-code` and `observed-features-plot-selected-region-code` if no longer needed.

These selectors are actually all used. There could be some optimization of the CSS, but that would require some restructuring to the component as well. Leaving be for now.